### ISSUE TITLE
Submit robotfindskitten to website robotfindskitten.org

### ISSUE THREAD

**Issue by mikegogulski** [CleverRaven/Cataclysm-DDA#26975]
TL;DR: Who added robotfindskitten to CDDA? Is there a link to the commit/PR?

Game version: master

Operating system: all

Tiles or curses: both

Mods active: none
 
### Expected behavior
CDDA's implementation of robotfindskitten is proudly listed among the ports and implementations at http://robotfindskitten.org/aw.cgi?main=software.rfk

### Actual behavior
CDDA's implementation of robotfindskitten is sadly MISSING from robotfindskitten.org :'( 

### Steps to reproduce the behavior
- Visit http://robotfindskitten.org/aw.cgi?main=software.rfk
- Notice that CDDA is not among the listed implementations
- Weep

### Proposed fix:
- Collect the necessary information to submit CDDA's robotfindskitten to the website via the instructions at http://robotfindskitten.org/aw.cgi?main=submit.rfk
- Submit said information
- Wait some time
- Celebrate upon receiving confirmation from the robotfindskitten consortium that CDDA's implementation has been included on the website

### Why the devil is this a Github issue?
I would send the details required immediately to resolve this important issue myself, but section 4 of the submission template wants authorship information which I am unable to locate. https://github.com/CleverRaven/Cataclysm-DDA/commit/a8f2d91f27e17d6aa6e7c60b1679fa7721d75431#diff-6f7cc7c541f3d7d97ad73c5604183e37 appears to be the first mention of the game in the source tree, but since this is a file move I am unable to trace back further than that to determine who submitted the robotfindskitten code.

The in-game credits for robotfindskitten (http://cddawiki.chezzo.com/cdda_wiki/index.php?title=Handheld_game_system) say "ported to CDDA gaming system by a nutcase", which may not be acceptable attribution.

### Draft robotfindskitten.org submission template:
To: rfk@robotfindskitten.org

Subject: robotfindskitten port submission: Cataclysm: Dark Days Ahead

1. port architecture

Cataclysm: Dark Days Ahead
            
 2. port "blurb"

Cataclysm: Dark Days Ahead, or CDDA, is an open-source survival horror roguelike video game, implemented in C++. During gameplay, the player has the opportunity to find Handheld Game Systems (http://cddawiki.chezzo.com/cdda_wiki/index.php?title=Handheld_game_system) and Game Watches which allow them to play, among other games, robotfindskitten. Finding kitten gives the player the `Found kitten <3` buff for a period of time, substantially boosting their morale amidst the zombie apocalypse.

3. The port's (optional) homepage. (The URL for  a page *you* created about your port.)

TBD. This should probably be one or the other of a link to https://cataclysmdda.org/ and/or the original robotfindskitten commit on Github or whatever.

4. credits for developers; this includes ALL of the following:
  a. personal name (doesn't have to be real)
  b. nickname (or personal name if no nickname)
  c. an email address to be posted on our site (or none)
  d. optional personal home page (not the project homepage)
   - see our devteam page for example usage of this information.

TBD

SOFTWARE:

4. a precompiled-binary (if possible)

CDDA release binaries are available at http://en.cataclysmdda.com/ . robotfindskitten was first added in version 0.8 Romero.

5. a sourcecode archive under a permissive license like the GPL, BSD, etc.

https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/iuse_software_kitten.cpp

Cataclysm is licensed under the Creative Commons Attribution-ShareAlike 3.0 Unported License.

SCREENSHOTS:

6. AT LEAST one screenshot of your port if possible. Please attach the screenshot to an email -- do not refer to screenshots available on the homepage.

TODO

Note to self: Rycon's sequence beginning at https://youtu.be/eYIeq98Ec4M?list=PLxIQ9fpV90xMG4FheSs0FGsORme8IayF0&t=657 demonstrates playing robotfindskitten on a Game Watch after slaughtering two Zombie Children, and the subsequent morale gain from the win.

**Comment by OzoneH3** [comment]
Implemented by @atomicdryad in #2894

**Comment by ZhilkinSerg** [comment]
And it is probably based upon http://robotfindskitten.org/aw.cgi?main=software.rfk#PDCurses.

**Comment by Night-Pryanik** [comment]
Issue could be closed then?
